pages/community/user-groups.html

```diff
@@ -44,6 +44,7 @@ <h1 class="intro-title">User Groups</h1>
 				'Indonesia': [-2.548926, 118.0148634],
 				'Italy': [42.504154, 12.646361],
 				'Iran': [32.4207423, 53.6830157],
+				'Iraq': [33.223191, 43.679291],
```

Comment:
I think we should add a check and handle missing entries in the list more gracefully. We should still log an error, but we shouldn't break and stop rendering the remainder of the list if some country is missing.

Reply:
Added a check:
![image](https://github.com/godotengine/godot-website/assets/270928/246c03b1-e2a6-4938-861b-b291fbb3df1e)